### DISCUSSION TITLE
Rustdesk over NetBird

### DISCUSSION THREAD

**Discussion by Aqua1ung** [rustdesk/rustdesk#5800] [Feature Request]
So far Rustdesk does not seem to work over [NetBird VPN](https://github.com/netbirdio/netbird), hence it would be nice to ... make it work with NetBird.

Either way, many thanks for a wonderful piece of software.

**Comment by Aqua1ung** [comment]
It's quite possible that I do not know how to configure Rustdesk to work w/NetBird. Looks like I may have to fill out some details in the "ID/Relay Server" section of Rustdesk's settings. Anyone?

**Reply by dinger1986** [reply]
should be a case of using the "internal IP"

**Comment by rustdesk** [comment]
works well with tailscale https://twitter.com/gospaceport/status/1704959110930653472
